ruby/ql/lib/codeql/ruby/ast/internal/AST.qll

```diff
@@ -130,6 +134,7 @@ private module Cached {
     TDoBlock(Ruby::DoBlock g) { not g.getParent() instanceof Ruby::Lambda } or
     TElementReference(Ruby::ElementReference g) or
     TElse(Ruby::Else g) or
+    TElseSynth(Ast::AstNode parent, int i) { mkSynthChild(ElseKind(), parent, i) } or
```

Comment:
Should we rename the existing `TElse` to `TElseReal` and define `class TElse = TElseReal or TElseSynth;`?

Reply:
We could, but not sure if adding `class TElse = TElseReal or TElseSynth` is useful.

Reply:
I think it is nice for consistency.